src/mixed-line-bar-chart/hooks/use-mouse-hover.ts

```diff
@@ -134,7 +134,11 @@ export function useMouseHover<T>({
   };
 
   const onPopoverLeave = (event: React.MouseEvent) => {
-    if (!isHandlersDisabled && !plotRef.current!.svg.contains(event.relatedTarget as Node)) {
+    if (
```

Comment:
How about using `nodeContains` utility, similar to this PR: https://github.com/cloudscape-design/components/pull/1374

And avoid fixing the same issue over and over again?

Reply:
It moved to the component-toolkit, but still available and should be used to mitigate all such issues